ocs_ci/framework/conf/ocp_version/ocp-4.16-config.yaml

```diff
@@ -0,0 +1,14 @@
+---
+# Config file for nightly OCP 4.16
+RUN:
+  client_version: "4.16.0-0.nightly"
+DEPLOYMENT:
+  installer_version: "4.16.0-0.nightly"
+  terraform_version: "1.0.11"
+  # ignition_version can be found here
+  # https://docs.openshift.com/container-platform/4.7/post_installation_configuration/machine-configuration-tasks.html#machine-config-overview-post-install-machine-configuration-tasks
+  ignition_version: "3.2.0"
+ENV_DATA:
+  vm_template: "rhcos-415.92.202311241643-0-vmware.x86_64"
```

Comment:
Can you please add TODO Comment to replace with 4.16 template once we have?

Reply:
done